solax/units.py

```diff
@@ -0,0 +1,28 @@
+""" Units and different measrement types"""
+from enum import Enum
+from typing import NamedTuple
+
+
+class Units(Enum):
+    """All known Units."""
+
+    W = "W"
+    KWH = "kWh"
+    A = "A"
+    V = "V"
+    C = "°C"
+    HZ = "Hz"
+    PERCENT = "%"
+    UNKNOWN = ""
```

Comment:
We probably want to call this some variant of "none", since it's not undefined- we just know it doesn't have a unit.